script/copy-sync-extension.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
+
+import sys
+import os
+import os.path
+from shutil import copyfile
+
+# Copies //brave/components/brave_sync/extension =>
+#        //brave/browser/resources/
+
+def copy_sync_extension():
+  files = ['components/brave_sync/extension/background.js',
```

Comment:
if we're going to copy these I think we should use the gn copy task to copy them into the output dir

Reply:
Never had met `gn copy task`. Will look and replace this python with copy task.